projects/common/src/constants/application-constants.ts

```diff
@@ -1,4 +1,4 @@
 export const enum ApplicationFeature {
   PageTimeRange = 'ui.page-time-range',
-  FeatureDefaultTimeRange = 'ui.feature-default-time-range-map'
+  TenantDefaultTimeRangeMap = 'ui.feature-default-time-range-map'
```

Comment:
```suggestion
  DefaultTimeRangeMap = 'ui.feature-default-time-range-map'
```

Tenant is redundant in the context of the UI, since the whole UI is loaded in a single tenant context.